### DISCUSSION TITLE
Nv setup

### DISCUSSION THREAD

**Discussion by appelgriebsch** [LazyVim/LazyVim#42] [Show and tell]
Hey,

I made big progress on my custom LazyVim setup. You can find it here: https://github.com/appelgriebsch/Nv.

It offers LSP / DAP support for Rust and Java, and also make some small updates to Telescope and CMP plugins.

**Comment by folke** [comment]
Your config looks really nice! Very clean. Love it

I may pick some stuff to add to LazyVim.

What's the difference with `:bnext` and the specific keymaps for bufferline? I thought those were the same or it probably depends on sorting? Then I should probably use that as a default for LazyVim.

Was planning to add DAP as well, so may use parts of your config for that too if that's ok :)

**Reply by appelgriebsch** [reply]
I use scope ("tiagovla/scope.nvim") to make bufferline only show buffers belonging to the current active tab. AFAIK you then also need to use BufferCycle... commands to limit the list of buffers to the ones in the current tab.... otherwise you may just jump from one tab to the other accidentially

https://github.com/akinsho/bufferline.nvim#how-do-i-see-only-buffers-per-tab

EDIT: looks like the limitation to use BufferCycle... is no longer valid: https://github.com/tiagovla/scope.nvim#about

**Reply by folke** [reply]
Just added the correct keymaps when bufferline is enabled. Otherwise it will default to `bnext/bprev`. 

It's still needed if users apply a sorting to bufferline

**Comment by luxus** [comment]
hi @appelgriebsch your config looks great, i will copy some stuff as well!
lazyvim is really great in this regards that people can reuse a lot of the code of others and it still feels personal.
looking forward to add golang dap

greetings from zurich
